src/pages/blog/[...slug].tsx

```diff
@@ -60,8 +60,8 @@ const BlogPage: NextPage<PageProps> = ({ post, content }) => {
                 description="You'll be searching your own code in 10 minutes. You can run it self-hosted (all of your code stays local and secure)."
                 slimWidth={true}
                 cta1={{
-                    text: 'Try Sourcegraph now',
-                    link: '/get-started/self-hosted',
+                    text: 'Get free trial',
+                    link: 'https://signup.sourcegraph.com/',
```

Comment:
We won't be able to merge this until signup is ready. I'm thinking we should merge this into the `4.0-launch` branch as well. Thoughts?

Reply:
Right, OK I removed this change so this branch can go into main & it's easier for blog writers. We'll introduce the CTA change in #[5739](https://github.com/sourcegraph/about/issues/5739)

Reply:
Awesome, thanks Becca!